### ISSUE TITLE
Not possible to remove images in settings

### ISSUE THREAD

**Issue by MasterGroosha** [tootsuite/mastodon#7003]
For example, let's go to site settings:
```
Hero image
Displayed on the frontpage. At least 600x100px recommended. When not set, falls back to instance thumbnail
```

However it's not possible to "remove" image once set. There's no "Delete image" button. Once set, it's forever unless you dig into source code and purge it manually and I'm not sure it's possible at all.
I assume the same problem also exists with thumbnail for open graph, user's avatar and banner.
* * * *

- [*] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [*] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by Cassolotl** [comment]
This seems related to (but is not the same as) *Impossible to erase images in config #4458*.

**Comment by MasterGroosha** [comment]
Yeah, it looks similar and related, but that issue is:  
1) very old  
2) applies only to personal settings, not site settings.

These issues might be merged into one though.

**Comment by sternenseemann** [comment]
Fixed in #13192
